src/sourmash/sourmash_args.py

```diff
@@ -392,15 +392,22 @@ def load_pathlist_from_file(filename):
     "Load a list-of-files text file."
     try:
         with open(filename, 'rt') as fp:
-            file_list = [ x.rstrip('\r\n') for x in fp ]
-
-        if not os.path.exists(file_list[0]):
-            raise ValueError("first element of list-of-files does not exist")
+            temp_file_list = [ x.rstrip('\r\n') for x in fp ]
+        file_list = set(temp_file_list)
```

Comment:
suggestion - I usually prefer just using the same variable name, e.g. get rid of `temp_file_list` altogether:
```
file_list = set(file_list)
```
Because of the way Python works, the value on the left is only set after the expression on the right is evaluated (ok that's normal for many languages :) and then the value that was used on the right is dereferenced and garbage collected (that's Python!) This has the slight benefit that you don't have the `temp_file_list` variable hanging around uselessly, either.

Reply:
Okay! I'll fix that!